tests/microos/live_patching.pm

```diff
@@ -0,0 +1,100 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: smoke test for Live Patching in SLE Micro
+#           1) Get current kernel version
+#           2) Make sure the system has been registered previously
+#           3) Register Live Patching module if not done already
+#           4) Optionally add a kernel update repo
+#           5) Update system and reboot
+#           6) Get new kernel version and compare it
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+use utils qw(zypper_call);
+use version_utils qw(is_sle_micro);
+
+sub register_lp_module {
+    my $arch        = get_required_var('ARCH');
+    my $regcode     = get_required_var('SCC_REGCODE');
+    my $sle_version = get_var('VERSION_SLE', '15.3');
```

Comment:
is there any way to avoid this? for instance to get it from `SUSEConnect --list-extensions` or ...?

Reply:
Yes, I don't like it either. Good idea, I'll try to extract it from the output.

Reply:
I just found a nice way to do this which works like a charm:
`my ($sle_version) = $extensions =~ /sle-module-live-patching\/(.*)\/$arch/;`

But we still have 15.3 hardcoded in the autoyast profile... so I would go with job variable and I will also remove `15.3` from the profile.